testutils/testutils-kmp/src/commonMain/kotlin/androidx/kruth/MapSubject.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.kruth
+
+import kotlin.test.fail
+
+class MapSubject<K, V>(actual: Map<K, V>?) : Subject<Map<K, V>>(actual) {
+
+    /** Fails if the map is not empty. */
+    fun isEmpty() {
+        requireNonNull(actual) { "Expected to be empty, but was null" }
+
+        if (actual.isNotEmpty()) {
+            fail("Expected to be empty")
```

Comment:
Can you include the actual value in here, we don't yet match the error messages from Truth, but one nice this about Truth is that it includes the actual value.

suggest: `Expected to be empty, but was $actual`

Reply:
Makes sense! Just updated the PR